dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/ws/security/kerberos/Krb5HelperJdk11.java

```diff
@@ -57,8 +56,14 @@
     /**
      * We don't do anything with the process, but having it set allows us to only be activated by DS if criteria we set
      * about the Java version are met.
+     *
+     * JDK 1.8 and vendor IBM, using IBM JGSS/KRB5
+     * JDK 1.8 and vendor not IBM, using SUN JGSS/KRB5
+     * JDK greater than 1.8 are using SUN JGSS/KRB5
      */
-    @Reference(policy = ReferencePolicy.STATIC, target = "(&(java.specification.version>=1.8)(!(java.vendor=ibm corporation)))")
+    @Reference(policy = ReferencePolicy.STATIC,
+               target = "(|(&(java.specification.version=1.8)(!(java.vendor=ibm corporation)))(java.specification.version>=1.8))")
```

Comment:
I still think you should simplify it to:

```
@Reference(policy = ReferencePolicy.STATIC, target = "(|(java.specification.version>1.8)(!(java.vendor=ibm corporation)))")
```

Reply:
apparently you cannot do '>' you have to do '>=' 
https://docs.osgi.org/javadoc/r2/org/osgi/framework/Filter.html

so` java.specification.version>=9 `would be the best way we could write it as

Reply:
Ok so:

@Reference(policy = ReferencePolicy.STATIC, target = "(|(java.specification.version>=9.0)(!(java.vendor=ibm corporation)))")

Reply:
ok, simplify it!!!